Dockerfile

```diff
@@ -24,9 +24,6 @@ USER appuser
 COPY --chown=appuser:appuser package.json yarn.lock /app/
 RUN yarn && yarn cache clean --force
 
-USER root
```

Comment:
Was this section not needed?

Reply:
It is not needed, the build works without it.